src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -0,0 +1,24 @@
+{% trans_default_domain "locationview" %}
+
+{% if draftVersions %}
+    <div class="ez-list-toolbar">
+        <h2 class="ez-list-toolbar-label">{{ 'locationview.versions.draft.under.edit'|trans()|desc('Draft under edit') }}</h2>
+    </div>
+    {% include '@EzSystemsHybridPlatformUi/content/tabs/versions/table.html.twig' with {'versions': draftVersions} %}
```

Comment:
Thanks, it's been a while since using twig and after a quick look it does look like the function is the one to use 👍